draft-gharris-opsawg-pcap.xml

```diff
@@ -10,7 +10,7 @@
 
 <rfc category='info'
      ipr='trust200902'
-     docName="draft-gharris-opsawg-pcap">
+     docName="draft-gharris-opsawg-pcap-01">
 
```

Comment:
-01, or -00?  [These guidelines](https://www.ietf.org/standards/ids/guidelines/#7) say "If the document is a new one (i.e., a version of "00") ...".

Reply:
Yes ! Updated.

Reply:
There's already a -01 that presumably @mcr submitted, so maybe we should go to -02 now.